docker/Dockerfile

```diff
@@ -93,8 +90,11 @@ RUN \
            /usr/local/lib/libgtest* \
            /usr/local/share/aclocal/gtest* \
            /opt/fiware-orion \
+           /usr/share/cracklib \
```

Comment:
As discussed at skype, it would be good to add some comment above this line to explain a little bit. Something like the following suggestion:

"# We cannot remove the following using yum as yum uses hard dependencies and doing so will uninstall essential packages. However we have empirically check that rm these libs is fine"